json/src/main/scala/com/gu/contentapi/json/CirceDecoders.scala

```diff
@@ -16,8 +16,8 @@ object CirceDecoders {
   implicit final val decodeString: Decoder[String] = new Decoder[String] {
     final def apply(c: HCursor): Decoder.Result[String] = {
       val focus = c.focus
-      val fromStringOrLong = focus.asString.orElse(focus.asNumber.flatMap(_.toLong.map(_.toString)))
-      Xor.fromOption(fromStringOrLong, ifNone = DecodingFailure("String", c.history))
+      val maybeFromStringOrLong = focus.asString.orElse(focus.asNumber.flatMap(_.toLong.map(_.toString)))
+      maybeFromStringOrLong.map(Right(_)).getOrElse(Left(DecodingFailure("String", c.history)))
```

Comment:
There is a fromOption provided as part of the cats syntax (possibly also in 2.12 stdlib).